docker-for-ibm-cloud/load-balancer.md

```diff
@@ -138,23 +157,58 @@ To specify other or multiple ports, append them as follows:
 * Links HTTPS to port 444: `--label com.ibm.d4ic.lb.cert=certificate-common-name@HTTPS:444`
 * Links HTTPS to ports 444 and 8080: `--label com.ibm.d4ic.lb.cert=certificate-common-name@HTTPS:444,HTTPS:8080`
 
-**Set a health check path**: By default, the service load balancer sets a health check path to `/`. If the service cannot respond with a 200 message to a `GET` request on the `/` path, then include a health monitor path label when you create the service. For example:
+**Set a health check path when using SSL termination**:
+By default, the service load balancer sets a health check path to `/`. If the service cannot respond with a 200 message to a `GET` request on the `/` path, then you must include a health monitor path label when you create the service. For example:
 
 ```bash
---label com.ibm.d4ic.healthcheck.path=/demo/hello@443
+--label com.ibm.d4ic.healthcheck.path=/demo/hello@444
 ```
 
 When the route is published, the health check is set to the path that you specify in the label. Choose a path that can respond with a 200 message to a `GET` request.
 
-**Example command**: The following `docker service create` command expands on the example from the [previous section](#access-a-service-with-the-service-load-balancer) to create a demo service that is exposed on a different port than the default and includes a health check path. It is based on the `vfarcic/go-demo` image.
+#### Example command for HTTPS
 
-```bash
-$ docker service create --name go-demo \
-  -e DB=go-demo-db \
-  --network go-demo \
-  --publish 8080:8080 \
-  --replicas 3 \
-  --label com.ibm.d4ic.lb.cert=certificate-common-name@HTTPS:8080 \
-  --label com.ibm.d4ic.healthcheck.path=/demo/hello@8080 \
-  vfarcic/go-demo
-```
+The following `docker service create` command expands on the example from the [previous section](#access-a-service-with-the-service-load-balancer) to create a demo service that is exposed on a different port than the default and includes a health check path.
+
+Before you begin:
+
+1. Log in to [IBM Cloud infrastructure](https://control.softlayer.com/).
+
+2. [Add or an import an SSL certificate](https://knowledgelayer.softlayer.com/topic/ssl-certificates) to use. In your infrastructure account, you can access the page from **Security** > **SSL** > **Certificates**.
+
+3. Note the certificate **Common Name**.
+
+Steps:
+
+1. Connect to your Docker EE for IBM Cloud swarm. Navigate to the directory where you [downloaded the UCP credentials](administering-swarms.md#download_client_certificates) and run the script. For example:
+
+   ```bash
+   $ cd filepath/to/certificate/repo && source env.
+   ```
+
+2. Create the service that you want to expose by using the `docker service create` [command](/engine/reference/commandline/service_create/). For example:
+
+   ```bash
+   $ docker service create --name nginx-test \
+     --publish 444:80 \
```

Comment:
You say above that the service listens on both port 444 and port 80, but this service creation statement does not accomplish that...

Reply:
Good catch! I've made the introductory write up more generic because it was confusing and users might think they could only use those ports.